### PR TITLE
Us/13: Add Event in Calendar from DayView/WeekView page

### PR DESCRIPTION
**Pull request by thclmnt**
Allow application to access calendar to add event from long pressing a class in DayView & WeekView page.

Notable changes:
- Added permissions for Android & iOS in app.json
- Added a long press event in CourseRow component
- Created a new modal/popup with new translations to confirm wanting to add event to calendar
- Handled app permissions to trigger only when user wants to add an event
- Added Toast to notify if event was successfully added to calendar or not
- Fixed a bug for Toast not working properly without `react-native-root-siblings` package in newer versions of React Native
- Added more information in FetchManager for events to be created with proper date

### CODE REVIEW

File: utils/translations/fr.js [hunk 1]
```diff
@@ -19,7 +24,8 @@ export default {
 	ERROR_WITH_CODE: 'Le serveur a renvoyé une erreur $-',
 	ERROR_WITH_MESSAGE: 'Erreur : $-',
 	FILTERS: 'Filtres',
-	FILTERS_CONFIRMATION: 'Voulez vous filtrer cette UE? Vous pouvez toujours annuler cette action dans les paramètres.',
+	FILTERS_CONFIRMATION:
+		'Voulez vous filtrer cette UE? Vous pouvez toujours annuler cette action dans les paramètres.',
```
Comment (AamuLumi) [inline utils/translations/fr.js:28]: **Voulez-vous filter cette UE ? ...**

File: utils/translations/fr.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 export default {
 	ABOUT: 'À propos',
+	ADD_TO_CALENDAR: "Ajouter l'événement au calendrier",
+	ADD_TO_CALENDAR_DESCRIPTION: 'Voulez vous ajouter l\'événement "$-" au calendrier?',
+	ADD_TO_CALENDAR_DONE: 'Événement ajouté !',
+	ADD_TO_CALENDAR_PERMISSIONS:
+		"L'application a besoin des permissions pour ajouter des événement au calendrier",
```
Comment (AamuLumi) [inline utils/translations/fr.js:7]: **pour ajouter des événements au ...**

**Review by AamuLumi — CHANGES_REQUESTED** [review]

File: utils/translations/fr.js (hunk 1 above)
Comment (thclmnt) [reply utils/translations/fr.js]: done

File: utils/translations/fr.js (hunk 2 above)
Comment (thclmnt) [reply utils/translations/fr.js]: done

**Review by AamuLumi — APPROVED** [review]

**Pull request merged**